subsys/zigbee/osif/zb_nrf_crypto.c

```diff
@@ -105,7 +105,7 @@ zb_uint32_t zb_random_seed(void)
 void zb_osif_aes_init(void)
 {
 #if CONFIG_CRYPTO_NRF_ECB
-	dev = device_get_binding(CONFIG_CRYPTO_NRF_ECB_DRV_NAME);
+	dev = device_get_binding(DT_LABEL(DT_INST(0, nordic_nrf_ecb)));
 	__ASSERT(dev, "Crypto driver not found");
```

Comment:
Might as well avoid calling `device_get_binding` if we can:

```
dev = DEVICE_DT_GET(DT_INST(0, nordic_nrf_ecb));
__ASSERT(device_is_ready(dev), "Crypto driver not found");
```

Using DEVICE_DT_GET or some affiliated function is basically always better than calling device_get_binding() if it's possible. It's faster since there's no loop over the devices, and it fails at link time instead of runtime if the device does not exist.